libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
```

Comment:
function should be written like this.stockPickerForm.valueChanges.subscribe(() => {
      this.fetchQuote();
    }); 
other wise fetchQuote not able to read property 'stockPickerForm' because of scope issue